library/Notifications/Model/Channel.php

```diff
@@ -45,4 +47,37 @@ public function getDefaultSort()
     {
         return ['name'];
     }
+
+    public function createRelations(Relations $relations)
+    {
+        $relations->hasMany('contact', Contact::class)->setJoinType('LEFT');
+        $relations->hasMany('incident_history', IncidentHistory::class)->setJoinType('LEFT');
+        $relations->hasMany('rule_escalation_recipient', RuleEscalationRecipient::class)->setJoinType('LEFT');
+    }
+
+    /**
+     * Fetch and map all the configured channel types to a key => value array
+     *
+     * @return array<string>
```

Comment:
Since the keys are not positions, please include them here with `int, ` and document them as being ids